modules/ROOT/pages/installation/self-signed-certificate.adoc

```diff
@@ -0,0 +1,66 @@
+:description: This section contains instructions to generate a self-signed certificate for test purposes.
+= Self-Signed Certificate Generation
+
+[IMPORTANT]
+====
+Self-signed certificates are not recommended to be used in production environments.
+For production environments, it is advisable to use a trusted certificate issuer.
+This section outlines a practical way to generate a self-signed certificate for test and demo purposes.
+====
+
+The NOM Server contains a utility to generate a self-signed certificate suitable for your environment.
+
+To invoke the utility and get help on usage, navigate to the NOM Server folder and execute the following command:
+
+[source, terminal, role=noheader]
+----
+$> java -jar ./lib/server.jar ssc --help
+
+utility to generate self signed certificate for TEST purposes only
+
+sample usage:
+
+to generate a self signed certificate for 'localhost', which could either be
+accessed through DNS names of 'localhost.localdomain' or 'my.custom.domain', or
+with IP addresses of '127.0.0.1' or '192.168.100.5';
+
+$> java -jar ./lib/server.jar ssc -n localhost -o /tmp -p changeit \
+   -d localhost.localdomain,my.custom.domain \
+   -i 127.0.0.1,192.168.100.5
+
+it will create /tmp/localhost.cer and /tmp/localhost.pfx files upon completion.
+
+options (required options marked with *):
+  -d, --dns=<dns>[,<dns>...]
+                            list of dns names you will use to access the
```

Comment:
```suggestion
                            list of dns names to use to access the
```

Reply:
I am not sure this conveys exactly the right meaning. The user is supplying the dns names that they will use to access the server. 

The suggested change implies that somehow this will _cause_ the dns names to be used

Maybe something like : 
`list of dns names that are used to access the`